core/tracker/tracker.go

```diff
@@ -67,11 +67,11 @@ type Tracker struct {
 	deadliner core.Deadliner
 	quit      chan struct{}
 
-	// failedDutyReporter instruments the duty. It ignores non-failed duties.
-	failedDutyReporter func(core.Duty, bool, string, string)
+	// failedDutyReporter instruments duty failures.
```

Comment:
discussion: what's the preferred approach when declaring function literals? omitting the names of the parameters or explicitly naming the parameters?

Reply:
only explicitly name parameters if it isn't obvious from the types what they mean. In this case `bool, string, string` doesn't indicate what they mean.